articles/iot-edge/production-checklist.md

```diff
@@ -316,6 +316,20 @@ You can do so in the **createOptions** of each module. For example:
 
 For the most efficient IoT Edge deployment scenario, consider integrating your production deployment into your testing and CI/CD pipelines. Azure IoT Edge supports multiple CI/CD platforms, including Azure DevOps. For more information, see [Continuous integration and continuous deployment to Azure IoT Edge](how-to-continuous-integration-continuous-deployment.md).
 
+
+### Each IoT Edge device should have its own unique device connection string 
+
+Please make sure unique device connection string is **not** inadvertently **shared** between 2 or more separate IoT Edge devices.
+IoT Edges devices sharing the device connection string may seem to work properly but it can lead to following issues before condition is detected and fixed
+
+- 1.)	Huge spike in module and device connects/disconnects
+- 2.)	Huge spike in module D2C Get Twin operations
```

Comment:
We reserve ordered lists for ordered processes. Please switch this to an unordered list. For example: 

```
* Item 1
* Item 2
```